drivers/ieee802154/Kconfig.b91

```diff
@@ -5,7 +5,8 @@
 
 menuconfig IEEE802154_TELINK_B91
 	bool "Telink B91 RF driver"
-	depends on NETWORKING
+	default y
+	depends on NETWORKING && DT_HAS_TELINK_B91_ZB_ENABLED
```

Comment:
seems like NETWORKING should be up a level and not at driver level??

Reply:
moved one level up